model/src/main/java/uk/gov/hmcts/fees2/register/data/model/Fee.java

```diff
@@ -149,9 +150,12 @@ public String toString() {
             ", unspecifiedClaimAmount=" + unspecifiedClaimAmount +
             ", creationTime=" + creationTime +
             ", lastUpdated=" + lastUpdated +
+            ", keyword='" + keyword + '\'' +
+            ", feeCodeHistories=" + feeCodeHistories +
```

Comment:
Is there any requirement to change the toString() implementation ?

Reply:
since added new field, updated toString() implementation accordingly

Reply:
My bad, copy pasted the same comment. If you see there's no reasonForUpdate field here, I've removed it already. There's no change in toString() method after that, apart from just rearrangement in lines....